src/test/function/Sql.json

```diff
@@ -0,0 +1,27 @@
+{
+  "bindings": [
+    {
+      "type": "sql",
+      "direction": "in",
+      "commandType": "text",
+      "connectionStringSetting": "SqlConnectionString",
+      "commandText": "SELECT * FROM dbo.Products",
+      "name": "result"
```

Comment:
What does this `name` represent?

Reply:
That's the base binding metadata name : https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/function.json#L6

In this case it corresponds to the variable/property name the binding is associated with.